jina/drivers/reduce.py

```diff
@@ -20,7 +20,7 @@ class ReduceAllDriver(BaseRecursiveDriver):
         It uses the last request as a reference.
     """
 
-    def __init__(self, traversal_paths: Tuple[str] = ('c',), *args, **kwargs):
+    def __init__(self, traversal_paths: Tuple[str] = ('m',), *args, **kwargs):
```

Comment:
Would this affect other examples? I see some failing unit tests and integration tests (and running them on master works)

Reply:
It will affect whenever a `ReduceAllDriver` is used that is not used to accumulate `matches`. So either we change this or we change the `_merge_all` in the resources, which I will do now.

Reply:
now u can see the alternative

Reply:
We could consider in the future adding different `aliases` depending on the reduction to be done, but now this would be the best option

Reply:
I agree

Reply:
what about this new alias?

Reply:
will need to add a test